### PR TITLE
Update CCP info for Cloud Backups directory: infodev-2693

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-bfba845b26/how-to/)

**Comment by RackerKate** [comment]
@cyrichardson For your review. I'm not sure how this all will work.

**Comment by cyrichardson** [comment]
@RackerKate In my queue.

**Comment by cyrichardson** [comment]
@RackerKate Reviewing this one now.

File: content/cloud-backup/cloud-backup-faq.md [hunk 1]
```diff
@@ -153,15 +153,18 @@ See the section on [Retaining your Backup](/how-to/rackspace-cloud-backup-backup
...
 
-You set up your backups in the Rackspace Cloud Backup Control Panel.
+You set up your backups in the Rackspace [Cloud Control Panel](http://login.rackspace.com).
 
-If you need to create a new Backup, click the System you want to back up,
-and then the **Create Backup** button. If you already have a Backup and
+Select **Rackspace Cloud** from the **Select a Product** drop-down menu, then
+select **Backups**.
+
+If you need to create a new Backup, click the System that you want to back up,
```
Comment (cyrichardson) [inline content/cloud-backup/cloud-backup-faq.md]: new backup

File: content/cloud-backup/cloud-backup-faq.md [hunk 2]
```diff
@@ -153,15 +153,18 @@ See the section on [Retaining your Backup](/how-to/rackspace-cloud-backup-backup
...
+and then click **Create Backup**. If you already have a Backup and
```
Comment (cyrichardson) [inline content/cloud-backup/cloud-backup-faq.md]: a backup

File: content/cloud-backup/cloud-backup-faq.md [hunk 3]
```diff
@@ -153,15 +153,18 @@ See the section on [Retaining your Backup](/how-to/rackspace-cloud-backup-backup
...
 want to re-configure it, click the gear icon next to the Backup name and
```
Comment (cyrichardson) [inline content/cloud-backup/cloud-backup-faq.md]: backup name

File: content/cloud-backup/cloud-backup-faq.md [hunk 4]
```diff
@@ -153,15 +153,18 @@ See the section on [Retaining your Backup](/how-to/rackspace-cloud-backup-backup
 
 #### How do I configure my Rackspace Cloud Backups?
 
-You set up your backups in the Rackspace Cloud Backup Control Panel.
+You set up your backups in the Rackspace [Cloud Control Panel](http://login.rackspace.com).
```
Comment (cyrichardson) [inline content/cloud-backup/cloud-backup-faq.md]: the [Cloud Control Panel]

File: content/cloud-backup/cloud-backup-faq.md [hunk 5]
```diff
@@ -183,9 +186,9 @@ again, but the new auth token is the same as the old one.
 **Solution**
 
 Account administrators can manage permission levels in the User
```
Comment (cyrichardson) [inline content/cloud-backup/cloud-backup-faq.md]: **User**
(bold)

File: content/cloud-backup/cloud-backup-faq.md [hunk 6]
```diff
@@ -183,9 +186,9 @@ again, but the new auth token is the same as the old one.
...
-Management section of the Cloud Control Panel. Submit a request to your
-account administrator for **Full** access to your account or
-**Administrative** access to Cloud Files for your sub-user account.
+Management section of the [Cloud Control Panel](http://login.rackspace.com).
```
Comment (cyrichardson) [inline content/cloud-backup/cloud-backup-faq.md]: **Management**
(bold)

File: content/cloud-backup/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 7]
```diff
@@ -21,18 +21,27 @@ As a Managed Infrastructure customer who administers your own servers, this arti
...
+3. Select **Tickets > Ticket List**.
 
-1. In the [Rackspace Cloud Control Panel](https://mycloud.rackspace.com), click on **Backups > Systems**.
-2. From the list of Cloud Backup Systems, click the backup server name. 
+If there are no open support tickets about the server in question, attempt to
+establish a connection, as shown in the next section.
+
+## Establish connection
+
+1. Log in to the [Rackspace Cloud Control Panel](https://login.rackspace.com).
```
Comment (cyrichardson) [inline content/cloud-backup/cloud-backup-troubleshooting-on-windows-2012-servers.md]: the [Cloud Control Panel]

File: content/cloud-backup/cloud-backup-troubleshooting-on-windows-2012-servers.md [hunk 8]
```diff
@@ -59,13 +68,13 @@ If there are no open support tickets about the server in question, attempt to es
...
+
    c. Click the **OK** button at the bottom of the page.
-   
-   d. After the service restarts, check the Control Panel to ensure the service is running.
-   
+
+   d. After the service restarts, check the Cloud Control Panel to ensure the service is running.
+
 7. Perform one of the following:
    - If the agent is connected, contact Rackspace Technical Support.
```
Comment (cyrichardson) [inline content/cloud-backup/cloud-backup-troubleshooting-on-windows-2012-servers.md]: Rackspace Support

File: content/cloud-backup/cloud-backup-troubleshooting.md [hunk 9]
```diff
@@ -237,7 +237,7 @@ Following are other errors and problems that you might encounter.
 
 A registered sub-user is authorized for Cloud Backup but not for Cloud Files access. When this user attempts a backup, all requests to Cloud Files return a 403 error. The user attempts to authenticate again, but the new authentication token is the same as the old one.
 
-Account administrators can manage permission levels in the User Management section of the Cloud Control Panel. Submit a request to your account administrator for Full access to your account or Administrative access to Cloud Files for your sub-user account. Cloud Backup does not support Dedicated Users with Cloud access or Federated users.
+Account administrators can manage permission levels in the User Management section of the [Cloud Control Panel](http://login.rackspace.com). Submit a request to your account administrator for Full access to your account or Administrative access to Cloud Files for your sub-user account. Cloud Backup does not support Dedicated Users with Cloud access or Federated users.
```
Comment (cyrichardson) [inline content/cloud-backup/cloud-backup-troubleshooting.md]: **Full**
(bold)
**Administrative**
(bold)
(to be consistency with what's in previous article)

File: content/cloud-backup/rackspace-cloud-backup-system-actions.md [hunk 10]
```diff
@@ -15,7 +15,14 @@ product_url: cloud-backup
 
 **Warning**: The system actions are for users who are familiar with Rackspace Cloud Backup. Users with Managed Cloud Service Level accounts might want to contact their account manager before performing any of the following actions.
 
-This article describes the actions available in the **Rackspace Cloud Control Panel -> Backups -> Systems** selection.
+This article describes the actions that are available in the Backups section of the Rackspace [Cloud Control Panel](http://login.rackspace.com/).
```
Comment (cyrichardson) [inline content/cloud-backup/rackspace-cloud-backup-system-actions.md]: the [Cloud Control Panel]

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-0156e36631/how-to/). :bow:

File: content/cloud-backup/rackspace-cloud-backup-backup-actions.md [hunk 11]
```diff
@@ -18,12 +18,16 @@ This article describes the actions you can perform during a single back up.
 
 You can access the **Backup Actions** menu in the following ways:
 
--   Select **Cloud Control Panel -> Backups -> Backups**, and then click on the gear icon next to the backup name on the **Configured Backups** page.
+-   Log in the [Cloud Control Panel](http://login.rackspace.com), click the
```
Comment (catlook) [inline content/cloud-backup/rackspace-cloud-backup-backup-actions.md]: Log in to the ...

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4676c33c4e/how-to/).

**Pull request merged**